src/resources/views/fields/page_or_link.blade.php

```diff
@@ -13,6 +13,9 @@
     $field['options']['external_link'] = $field['options']['external_link'] ?? trans('backpack::crud.external_link');
 
     $field['pages'] = $field['pages'] ?? ($field['page_model'] ?? config('backpack.pagemanager.page_model_class'))::all();
+
+    $field['internal']['default'] = ($field['default_internal'] ? $field['default_internal'] : "");
```

Comment:
Why do we change from `$field['default_internal']` to `$field['internal']['default']` ? Are expecting to add more configuration keys insid `internal` ? If not please keep the same `$field['default_internal']`

Reply:
Hi @pxpm please suggest what can be better to do this?